packages/blocks/src/components/drag-handle.ts

```diff
@@ -398,6 +433,45 @@ export class DragHandle extends LitElement {
     );
   }
 
+  private _createDragMirror(
+    e: DragEvent,
+    draggingBlockElements: BlockComponentElement[]
+  ) {
+    const containerRect = this._container.getBoundingClientRect();
+    const rect = draggingBlockElements[0].getBoundingClientRect();
+
+    const dragMirror = new DragMirror();
+    dragMirror.offset.x = rect.left - containerRect.left - e.clientX;
+    dragMirror.offset.y = rect.top - containerRect.top - e.clientY;
+    dragMirror.style.width = `${rect.width}px`;
+    dragMirror.style.transform = `translate(${
+      rect.left - containerRect.left
+    }px, ${rect.top - containerRect.top}px)`;
+
+    const fragment = document.createDocumentFragment();
+    draggingBlockElements.forEach(e => {
+      const c = document.createElement('div');
+      c.classList.add('affine-block-element');
+      render(e.render(), c);
```

Comment:
The mode and other information will be lost if use `cloneNode`.

Reply:
This is a great idea of using render.
What I am worried is that creating lit elements will cause some issues, for example the same block id attr in DOM element being added to the page and may cause some selectors in blocksuite will fail, or having some unknown sideeffects because of creating new custom elements. But I guess we can fix any issue if we happen to find any in the future anyway.

Reply:
https://lit.dev/docs/api/templates/

Reply:
Drag mirror is generated only when dragging

Reply:
I think `data-block-id` can be allowed to have duplicates (though we should avoid it most of the times), and that's why we don't use the standard DOM `id` in this case.

Reply:
In practical tests, the main problem is that the mode data is lost if use `cloneNode`.